src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    commonBrowserIssue: 'Seu browser {virtualclass1} {virtualclass2} não é suportado, Use a última versão do Chrome ou Firefox.',
+    congreainchrome: 'Seu browser é compatível.',
+    confirmCancel: 'Não',
+    confirmOk: 'Sim',
+    connectionClose: 'Conexão está fechada ({virtualclass1}).',
+    Controls: 'Controles',
+    coursePoll: 'Enquete a nível do curso',
+    Cpoll: 'Enquete do curso',
+    Cquiz: 'Fechar quiz',
+    createnewpoll: 'Criar nova enquete',
+    Creator: 'Criador',
+    comment: 'Comentários',
+    dap: 'falso',
+    delete: 'Deletar',
+    delblank: 'Remover a opção em branco',
+    Delete: 'Deletar',
+    DevicesNotFoundError: 'Webcam não está disponível, ou não está conectada corretamente.',
+    deletepopup: 'Tem certeza que deseja deletar ?',
+    disable: 'Desabilitar',
+    disabled: 'desabilitado',
+    disableAllVideo: 'Desabilitar todo vídeo',
+    dltDisabled: 'Pode ser deletado pelo criador da enquete',
+    documentShare: 'Compartilhar documento',
+    DocumentSharing: 'Compartilhamento de documento',
+    DocumentSharedbHeading: 'Painel de documento',
+    docUploadSuccess: 'Documento enviado com sucesso',
+    download: 'Baixar',
+    downloadFile: 'Baixar arquivo',
+    downloadsession: 'Favor aguardar enquanto a gravação é baixada',
+    drawArea: 'Área de desenho',
+    dropfilehere: 'Solte os arquivos aqui',
+    duplicateUploadMsg: 'Este arquivo já existe, por favor envie um novo arquivo.',
+    edit: 'Editar',
+    editorCode: 'Editor de código',
+    edittitle: 'Editar título',
+    editorRichDisable: 'Modo de escrita',
+    editorRichEnable: 'Apenas leitura',
+    educator: 'Educador',
+    EHTMLPresentUrl: 'Inserir URL da apresentação HTML5',
+    enable: 'Habilitar',
+    enablehistory: '"Histórico do navegador" precisa estar habilitado durante a sessão.',
+    enableAllVideo: 'Habilitar todo vídeo',
+    enteryouryoutubeurl: 'Inserir URL de vídeo online ou YouTube',
+    etDisabledA: 'Não pode editar, enquete já foi tentada',
+    etDisabledCr: 'Pode ser editada pelo criador da enquete',
+    ETime: 'Tempo decorrido',
+    FF2: 'Acelerar x2',
+    FF8: 'Acelerar x8',
+    filenotsave: 'Seu arquivo não pode ser salvo.',
+    filetsaveTS: 'A sessão está pronta para salvar.',
+    Finish: 'Finalizar',
+    fitToPage: 'Ajustar à página',
+    fitToScreen: 'Ajustar à tela',
+    Freehand: 'Mão livre',
+    general: 'Geral',
+    Greport: 'Relatório de nota',
+    highBandWidthSpeed: 'Sua banda larga é boa o suficiente.',
+    httpsmissing: 'Apenas origens seguras(https) são permitidas para compartilhamento de tela.',
+    ieBrowserIssue: 'Internet Explorer não é suportado, Congrea suporta completamente Chrome  e Firefox.',
+    ios7support: 'Para Apple, Sala Virtual suporta iOS 8 e versões superiores.',
+    iosAudEnable: 'Toque aqui para habilitar áudio',
+    InternalError: 'Por favor, certifique-se que a mesma webcam não está sendo usada <br /> em múltiplos browsers ou múltiplas aplicações. ',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 141. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)